judo-runtime-core-validator/src/main/java/hu/blackbelt/judo/runtime/core/validator/DefaultPayloadValidator.java

```diff
@@ -68,11 +68,17 @@ public enum RequiredStringValidatorOption {
     public static final String VALIDATE_MISSING_FEATURES_KEY = "validateMissingFeatures";
     public static final String IGNORE_INVALID_VALUES_KEY = "ignoreInvalidValues";
 
+    public static final String VALIDATE_ROOT_MISSING_FEATURES_KEY = "validateRootMissingFeatures";
+
+    public static final String IS_ROOT_KEY = "isRootKey";
```

Comment:
Key postfix is for showing that it's used as a key in maps. I don't think key postfix in string value is needed in "isRootKey"

Reply:
isRootKey is used as a key in validationContext

Reply:
[8f52346](https://github.com/BlackBeltTechnology/judo-runtime-core/pull/199/commits/8f52346453e60d18a3545f74fc24452454c7068c)